### PR TITLE
Add a short organisation profile with a hiring call to action

### PR DESCRIPTION
**Pull request by purcell**
This content will appear on the org page at http://github.com/tweag. Obviously there's more we could put in here, but I figured something short and aimed at developers was the best starting point.

The banner is the same one we use for the youtube channel.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Hello from the Tweag team!
+
+![Scale your engineering power](./banner.jpg)
+
+At Tweag we love using open source methods and research ideas to
+improve software quality and reliability. You might have seen some of
```
Comment (arobertn) [inline profile/README.md]: maybe
```suggestion
At Tweag we love applying open source and research ideas in typed
functional programming and formal methods to improve software quality
and reliability. You might have seen some of
```
.. or something like that anyway, just to get the technology themes in there.

**Review by mboes — APPROVED** [review]
Good enough, though I'd be happier if the copy was more in line with what we say typically say in job posts. A pretty strong signal I hear from candidates is that the "functional, typed, immutable" spiel is frequently a reason to at least have a look at Tweag.

**Comment by mboes** [comment]
Here's Microsoft's profile README for inspiration: https://github.com/microsoft.

**Review by layus — APPROVED** [review]

**Comment by purcell** [comment]
Thanks for the comments, @arobertn and @mboes! I didn't want to pin things down to specific technologies, thinking that the pinned repos should be representative of this, and the copy overall was borrowed from what we used at the recent PhD fair. I've now added the composable/functional/typed/immutable spiel, and will go ahead and merge.

**Comment by purcell** [comment]
Banner image isn't showing up, despite displaying in the PR preview, and following the guidelines for image linking. I've seen caching issues in the past, so I'll give it a short while and if it doesn't clear up I'll try a different image URL format.

**Comment by arobertn** [comment]
It looks like it doesn't translate the img url. Giving an absolute "https://github.com/tweag/.github/raw/master/profile/banner.jpg" might work.

**Pull request merged**